### ISSUE TITLE
Crash on world generation with Quilt Loader

### ISSUE THREAD

**Issue by BogTheMudWing** [RedLime/SpeedRunIGT#74]
**Describe the bug**
mouseClicked/keyPressed event handler error when trying to create a new world.

**To Reproduce**
1. Create a new world
2. Click Create or press enter

**Expected behavior**
World generates

**Screenshots or logs**
```
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 2022-10-17 19:25:16
Description: keyPressed event handler

java.lang.NoSuchMethodError: 'net.fabricmc.loader.impl.game.GameProvider net.fabricmc.loader.impl.FabricLoaderImpl.getGameProvider()'
	at com.redlimerl.speedrunigt.timer.InGameTimerUtils.getMinecraftVersion(InGameTimerUtils.java:204)
	at com.redlimerl.speedrunigt.timer.InGameTimerUtils.convertTimelineJson(InGameTimerUtils.java:145)
	at com.redlimerl.speedrunigt.timer.InGameTimer.updateRecordString(InGameTimer.java:389)
	at com.redlimerl.speedrunigt.timer.InGameTimer.setPause(InGameTimer.java:622)
	at com.redlimerl.speedrunigt.timer.InGameTimer.start(InGameTimer.java:139)
	at net.minecraft.class_7196.handler$bfp000$onCreate(class_7196.java:527)
	at net.minecraft.class_7196.method_41889(class_7196.java)
	at net.minecraft.class_525.method_41847(class_525.java:326)
	at net.minecraft.class_7196.method_41892(class_7196.java:247)
	at net.minecraft.class_525.method_2736(class_525.java:310)
	at net.minecraft.class_525.method_19922(class_525.java:260)
	at net.minecraft.class_4185.method_25306(class_4185.java:43)
	at net.minecraft.class_4264.method_25404(class_4264.java:26)
	at net.minecraft.class_4069.method_25404(class_4069.java:65)
	at net.minecraft.class_437.method_25404(class_437.java:124)
	at net.minecraft.class_525.method_25404(class_525.java:408)
	at net.minecraft.class_309.method_1454(class_309.java:374)
	at net.minecraft.class_437.method_25412(class_437.java:489)
	at net.minecraft.class_309.method_1466(class_309.java:370)
	at net.minecraft.class_309.method_22678(class_309.java:462)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_309.method_22676(class_309.java:462)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:222)
	at net.minecraft.class_310.method_1523(class_310.java:1208)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:423)
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:76)
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:24)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.redlimerl.speedrunigt.timer.InGameTimerUtils.getMinecraftVersion(InGameTimerUtils.java:204)
	at com.redlimerl.speedrunigt.timer.InGameTimerUtils.convertTimelineJson(InGameTimerUtils.java:145)
	at com.redlimerl.speedrunigt.timer.InGameTimer.updateRecordString(InGameTimer.java:389)
	at com.redlimerl.speedrunigt.timer.InGameTimer.setPause(InGameTimer.java:622)
	at com.redlimerl.speedrunigt.timer.InGameTimer.start(InGameTimer.java:139)
	at net.minecraft.class_7196.handler$bfp000$onCreate(class_7196.java:527)
	at net.minecraft.class_7196.method_41889(class_7196.java)
	at net.minecraft.class_525.method_41847(class_525.java:326)
	at net.minecraft.class_7196.method_41892(class_7196.java:247)
	at net.minecraft.class_525.method_2736(class_525.java:310)
	at net.minecraft.class_525.method_19922(class_525.java:260)
	at net.minecraft.class_4185.method_25306(class_4185.java:43)
	at net.minecraft.class_4264.method_25404(class_4264.java:26)
	at net.minecraft.class_4069.method_25404(class_4069.java:65)
	at net.minecraft.class_437.method_25404(class_437.java:124)
	at net.minecraft.class_525.method_25404(class_525.java:408)
	at net.minecraft.class_309.method_1454(class_309.java:374)
	at net.minecraft.class_437.method_25412(class_437.java:489)
	at net.minecraft.class_309.method_1466(class_309.java:370)
	at net.minecraft.class_309.method_22678(class_309.java:462)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_309.method_22676(class_309.java:462)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)

-- Affected screen --
Details:
	Screen name: net.minecraft.class_525
Stacktrace:
	at net.minecraft.class_437.method_25412(class_437.java:489)
	at net.minecraft.class_309.method_1466(class_309.java:370)
	at net.minecraft.class_309.method_22678(class_309.java:462)
	at net.minecraft.class_1255.execute(class_1255.java:103)
	at net.minecraft.class_309.method_22676(class_309.java:462)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:222)
	at net.minecraft.class_310.method_1523(class_310.java:1208)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:423)
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:76)
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:24)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, continuity/default, continuity/glass_pane_culling_fix, VanillaTweaks_r778603.zip, replaymod_lang
Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2348)
	at net.minecraft.class_310.method_1514(class_310.java:787)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quiltmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:423)
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:76)
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:24)

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 836062368 bytes (797 MiB) / 1979711488 bytes (1888 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 16
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i5-12600K
	Identifier: Intel64 Family 6 Model 151 Stepping 2
	Microarchitecture: unknown
	Frequency (GHz): 3.69
	Number of physical packages: 1
	Number of physical CPUs: 10
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 3060
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2504
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2225
	Graphics card #1 name: Intel(R) UHD Graphics 770
	Graphics card #1 vendor: Intel Corporation (0x8086)
	Graphics card #1 VRAM (MB): 1024.00
	Graphics card #1 deviceId: 0x4680
	Graphics card #1 versionInfo: DriverVersion=30.0.101.1273
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 29469.64
	Virtual memory used (MB): 19650.70
	Swap memory total (MB): 13312.00
	Swap memory used (MB): 174.02
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Loaded Shaderpack: (off)
	Quilted Fabric API: !! WARNING !! This instance is using Fabric API modules re-implemented by QSL. If the issue comes from Quilted Fabric API, DO NOT report to Fabric; report them to Quilt instead!
	Quilt Mods: 
		advanced_runtime_resource_pack: Runtime Resource Pack 0.6.2
		architectury: Architectury 6.2.46
		authme: Auth Me 4.1.0
		betterf3: BetterF3 1.3.2
		bobby: Bobby 4.0.0
		borderlessmining: Borderless Mining 1.1.5+1.19
		citresewn: CIT Resewn 1.1.2+1.19.2
		citresewn-defaults: CIT Resewn: Defaults 1.1.2+1.19.2
		cloth-api: Cloth API 4.0.65
		cloth-basic-math: cloth-basic-math 0.6.1
		cloth-client-events-v0: Cloth Client Events v0 4.0.65
		cloth-common-events-v1: Cloth Common Events v1 4.0.65
		cloth-config: Cloth Config v8 8.2.88
		cloth-scissors-api-v1: Cloth Scissors API v1 4.0.65
		cloth-utils-v1: Cloth Utils v1 4.0.65
		com_github_llamalad7_mixinextras: MixinExtras 0.0.11
		com_logisticscraft_occlusionculling: occlusionculling 0.0.6-SNAPSHOT
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		com_typesafe_config: config 1.4.2
		continuity: Continuity 2.0.1+1.19
		craftpresence: CraftPresence 1.9.2
		dynamicfps: Dynamic FPS 2.2.0
		enhancedblockentities: Enhanced Block Entities 0.7.1+1.19
		entityculling: EntityCulling-Fabric 1.5.2-mc1.19
		fabrishot: Fabrishot 1.8.0
		ferritecore: FerriteCore 4.2.1
		fvt: Flour's Various Tweaks 1.13.0-mc1.19.2
		indium: Indium 1.0.9+mc1.19.2
		inspecio: Inspecio 1.6.0+1.19
		io_leangen_geantyref_geantyref: geantyref 1.3.13
		iris: Iris 1.3.1
		java: OpenJDK 64-Bit Server VM 17
		lambdynlights: LambDynamicLights 2.1.2+1.19
		lazydfu: LazyDFU 0.1.3
		libzoomer: LibZoomer 0.5.0+1.19
		litematica: Litematica 0.12.4
		lithium: Lithium 0.8.3
		malilib: MaLiLib 0.13.0
		midnightlib: MidnightLib 0.5.2
		minecraft: Minecraft 1.19.2
		mm: Manningham Mills 2.1
		modmenu: Mod Menu 4.0.6
		nbttooltip: NBT Tooltip 1.7.0
		nochatreports: No Chat Reports 1.19.2-v1.13.6
		notenoughanimations: NotEnoughAnimations 1.6.1
		ok_zoomer: Ok Zoomer 5.0.0-beta.9+1.19
		org_anarres_jcpp: jcpp 1.4.14
		org_codehaus_groovy_groovy: groovy 3.0.11
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.11
		org_joml_joml: joml 1.10.4
		org_spongepowered_configurate-core: configurate-core 4.1.2
		org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		org_yaml_snakeyaml: snakeyaml 1.31
		phosphor: Phosphor 0.8.1
		pride: Pride Lib 1.1.2+1.19
		quilt_advancement: Quilt Advancement API 3.0.0-beta.18+1.19.2
		quilt_base: Quilt Base API 3.0.0-beta.18+1.19.2
		quilt_biome: Quilt Biome API 3.0.0-beta.18+1.19.2
		quilt_block_content_registry: Quilt Block Content Registry API 3.0.0-beta.18+1.19.2
		quilt_block_entity: Quilt Block Entity API 3.0.0-beta.18+1.19.2
		quilt_block_extensions: Quilt Block Extensions API 3.0.0-beta.18+1.19.2
		quilt_client_command: Quilt Client Command API 3.0.0-beta.18+1.19.2
		quilt_command: Quilt Command API 3.0.0-beta.18+1.19.2
		quilt_crash_info: Quilt Crash Info 3.0.0-beta.18+1.19.2
		quilt_datafixerupper: Quilt DataFixerUpper API 3.0.0-beta.18+1.19.2
		quilt_dimension: Quilt Dimension API 3.0.0-beta.18+1.19.2
		quilt_entity: Quilt Entity API 3.0.0-beta.18+1.19.2
		quilt_entity_events: Quilt Entity Events 3.0.0-beta.18+1.19.2
		quilt_entity_multipart: Quilt Multipart Entity API 3.0.0-beta.18+1.19.2
		quilt_entity_networking: Quilt Entity Networking API 3.0.0-beta.18+1.19.2
		quilt_item_content_registry: Quilt Item Content Registry API 3.0.0-beta.18+1.19.2
		quilt_item_extension: Quilt Item Extension API 3.0.0-beta.18+1.19.2
		quilt_item_group: Quilt Item Group API 3.0.0-beta.18+1.19.2
		quilt_item_setting: Quilt Item Setting API 3.0.0-beta.18+1.19.2
		quilt_lifecycle_events: Quilt Lifecycle Events 3.0.0-beta.18+1.19.2
		quilt_loader: Quilt Loader 0.17.5-beta.9
		quilt_networking: Quilt Networking 3.0.0-beta.18+1.19.2
		quilt_points_of_interest: Quilt Points of Interest 3.0.0-beta.18+1.19.2
		quilt_recipe: Quilt Recipe API 3.0.0-beta.18+1.19.2
		quilt_registry: Quilt Registry API 3.0.0-beta.18+1.19.2
		quilt_registry_entry_attachment: Quilt Registry Entry Attachment 3.0.0-beta.18+1.19.2
		quilt_resource_loader: Quilt Resource Loader 3.0.0-beta.18+1.19.2
		quilt_screen: Quilt Screen API 3.0.0-beta.18+1.19.2
		quilt_surface_rule: Quilt Surface Rule API 3.0.0-beta.18+1.19.2
		quilt_tags: Quilt Tags API 3.0.0-beta.18+1.19.2
		quilt_tooltip: Quilt Tooltip API 3.0.0-beta.18+1.19.2
		quilt_vehicle: Quilt Vehicle API 3.0.0-beta.18+1.19.2
		quilt_villager: Quilt Villager API 3.0.0-beta.18+1.19.2
		quilted_fabric_api: Quilted Fabric API 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_api_base: Quilted Fabric API Base 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_api_lookup_api_v1: Quilted Fabric API Lookup API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_biome_api_v1: Quilted Fabric Biome API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_blockrenderlayer_v1: Quilted Fabric BlockRenderLayer Registration (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_client_tags_api_v1: Quilted Fabric Client Tags 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_command_api_v1: Quilted Fabric Command API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_command_api_v2: Quilted Fabric Command API (v2) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_commands_v0: Quilted Fabric Commands (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_containers_v0: Quilted Fabric Containers (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_content_registries_v0: Quilted Fabric Content Registries (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_convention_tags_api_v1: Quilted Fabric Convention Tags API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_crash_report_info_v1: Quilted Fabric Crash Report Info (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_data_generation_api_v1: Quilted Fabric Data Generation API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_dimensions_v1: Quilted Fabric Dimensions API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_entity_events_v1: Quilted Fabric Entity Events (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_events_interaction_v0: Quilted Fabric Events Interaction (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_events_lifecycle_v0: Quilted Fabric Events Lifecycle (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_game_rule_api_v1: Quilted Fabric Game Rule API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_item_api_v1: Quilted Fabric Item API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_item_groups_v0: Quilted Fabric Item Groups (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_key_binding_api_v1: Quilted Fabric Key Binding API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_keybindings_v0: Quilted Fabric Key Bindings (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_lifecycle_events_v1: Quilted Fabric Lifecycle Events (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_loot_api_v2: Quilted Fabric Loot API (v2) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_loot_tables_v1: Quilted Fabric Loot Tables (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_message_api_v1: Quilted Fabric Message API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_mining_level_api_v1: Quilted Fabric Mining Level API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_models_v0: Quilted Fabric Models (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_networking_api_v1: Quilted Fabric Networking API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_networking_v0: Quilted Fabric Networking (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_object_builder_api_v1: Quilted Fabric Object Builder API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_particles_v1: Quilted Fabric Particles (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_registry_sync_v0: Quilted Fabric Registry Sync (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_renderer_api_v1: Quilted Fabric Renderer API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_renderer_indigo: Quilted Fabric Renderer - Indigo 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_renderer_registries_v1: Quilted Fabric Renderer Registries (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_rendering_data_attachment_v1: Quilted Fabric Rendering Data Attachment (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_rendering_fluids_v1: Quilted Fabric Rendering Fluids (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_rendering_v0: Quilted Fabric Rendering (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_rendering_v1: Quilted Fabric Rendering (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_resource_conditions_api_v1: Quilted Fabric Resource Conditions API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_resource_loader_v0: Quilted Fabric Resource Loader (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_screen_api_v1: Quilted Fabric Screen API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_screen_handler_api_v1: Quilted Fabric Screen Handler API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_sound_api_v1: Quilted Fabric Sound API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_textures_v0: Quilted Fabric Textures (v0) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_transfer_api_v1: Quilted Fabric Transfer API (v1) 4.0.0-beta.15+0.63.0-1.19.2
		quilted_fabric_transitive_access_wideners_v1: Quilted Fabric Transitive Access Wideners (v1) 4.0.0-beta.15+0.63.0-1.19.2
		replaymod: Replay Mod 1.19.1-2.6.8
		skiptransitions: Skip Transitions 1.3.2+1.18.x
		slight-gui-modifications: 'Slight' GUI Modifications 2.3.0
		sodium: Sodium 0.4.4+build.18
		speedrunigt: SpeedRunIGT 12.1+1.19.2
		spruceui: SpruceUI 4.0.0+1.19
		timechanger: Time Changer 1.2.0
		tooltipfix: ToolTip Fix 1.0.5-1.19
		viafabric: ViaFabric 0.4.9+19-main
		viafabric-mc119: ViaFabric for 1.19 0.4.9+19-main
		viaversion: ViaVersion 4.4.3-SNAPSHOT
		worldedit: WorldEdit 7.2.12+6240-87f4ae1
		worldeditcui: WorldEditCUI 1.19.2+01-SNAPSHOT
	Launched Version: quilt-loader-0.17.5-beta.9-1.19.2
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: NVIDIA GeForce RTX 3060/PCIe/SSE2 GL version 3.2.0 NVIDIA 522.25, NVIDIA Corporation
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'quilt'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, continuity/default, continuity/glass_pane_culling_fix, file/VanillaTweaks_r778603.zip
	Current Language: English (US)
	CPU: 16x 12th Gen Intel(R) Core(TM) i5-12600K
```
**Versions info :**
 - MC Version: 1.19.2
 - SpeedRunIGT Version: SpeedRunIGT-12.1+1.19.2
 - Fabric Loader Version: quilt-loader-0.17.5-beta-9.1.19.2

**Additional context**
If this is an issue on the side of QFAPI I can report it there instead.

**Comment by RedLime** [comment]
Will be fixed in 12.2

**Comment by RedLime** [comment]
Fixed and Released in 12.2, I tested mc 1.19.2 with quilt loader 0.17.4
